.travis.yml

```diff
@@ -1,9 +1,21 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
+    - 5.3
+    - 5.4
+    - 5.5
+    - 5.6
 
-before_script:
-    - curl -s http://getcomposer.org/installer | php
-    - php composer.phar install --dev
\ No newline at end of file
+matrix:
+    fast_finish: true
+    include:
+        - php: 5.6
+          env: SYMFONY_VERSION='2.3.*'
+        - php: 5.6
+          env: SYMFONY_VERSION='2.7.*'
+
+install:
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/symfony=$SYMFONY_VERSION; fi
+    - composer install --no-interaction --prefer-source
```

Comment:
rather than using ``--prefer-source``, I suggest switching to the new docker-based infrastructure (which is a good idea for any project not relying on sudo), enabling caching of ``$HOME/.composer/cache`` between builds and letting Composer use dist archives which will then be cached and reused between builds.